plugin/local/__init__.py

```diff
@@ -0,0 +1 @@
+from lsm.plugin.megaraid.megaraid import MegaRAID
```

Comment:
Why is this import here in the __init__.py, while the other two are in plugin/local/local.py?